### DISCUSSION TITLE
Recovered Of My Account

### DISCUSSION THREAD

**Discussion by queenwealthbuilder** [community/community#21440] [Other Feature Feedback, Questions, & Ideas]
<p>Hello.  I was able to recover my MIOTA account from Trinity to Firefly.  Just one problem though - my MIOTA balance is Zero.  That is incorrect!  How can I get my balance transferred over from Trinity.  Thanks!</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hi and welcome <a class="mention" href="/queenwealthbuilder">@queenwealthbuilder</a> ,</p>
<p>While all questions are welcome here, I don’t think you’ll get a proper answer here quickly. Would it be an idea to ask your question on a forum that is a little more specialized towards the technologies you’re asking about?</p>
